### PR TITLE
Split SDK into namespaces

### PR DESCRIPTION
**Pull request by thebrianchen**

### CODE REVIEW

File: src/api/nft-namespace.ts [hunk 1]
```diff
@@ -0,0 +1,646 @@
...
+
+    return {
+      contractAddress: response.contractAddress,
+      refreshState: parseReingestionState(response.reingestionState),
+      progress: response.progress
+    };
+  }
+}
+
+async function refresh(
```
Comment (deric-alchemy) [inline src/api/nft-namespace.ts:479]: Should we make this private? Assuming it's an internal method

Comment (thebrianchen) [reply src/api/nft-namespace.ts:479]: The function isn't exported and isn't a member function on the namespace class, so we're fine.

File: src/api/nft-namespace.ts [hunk 2]
```diff
@@ -0,0 +1,646 @@
...
+    return getNftFromRaw(response, contractAddress);
+  }
+
+  /**
+   * Get the NFT collection metadata associated with the provided parameters.
+   *
+   * @param contractAddress - The contract address of the NFT.
+   * @public
+   */
+  async getNftContractMetadata(contractAddress: string): Promise<NftContract> {
```
Comment (deric-alchemy) [inline src/api/nft-namespace.ts:80]: This should be getContractMetadata post-namespace!

Comment (thebrianchen) [reply src/api/nft-namespace.ts:80]: done

**Pull request merged**